app-modules/research/resources/views/results.blade.php

```diff
@@ -0,0 +1,54 @@
+{{--
+<COPYRIGHT>
+
+    Copyright © 2016-2025, Canyon GBS LLC. All rights reserved.
+
+    Advising App™ is licensed under the Elastic License 2.0. For more details,
+    see https://github.com/canyongbs/advisingapp/blob/main/LICENSE.
+
+    Notice:
+
+    - You may not provide the software to third parties as a hosted or managed
+      service, where the service provides users with access to any substantial set of
+      the features or functionality of the software.
+    - You may not move, change, disable, or circumvent the license key functionality
+      in the software, and you may not remove or obscure any functionality in the
+      software that is protected by the license key.
+    - You may not alter, remove, or obscure any licensing, copyright, or other notices
+      of the licensor in the software. Any use of the licensor’s trademarks is subject
+      to applicable law.
+    - Canyon GBS LLC respects the intellectual property rights of others and expects the
+      same in return. Canyon GBS™ and Advising App™ are registered trademarks of
+      Canyon GBS LLC, and we are committed to enforcing and protecting our trademarks
+      vigorously.
+    - The software solution, including services, infrastructure, and code, is offered as a
+      Software as a Service (SaaS) by Canyon GBS LLC.
+    - Use of this software implies agreement to the license terms and conditions as stated
+      in the Elastic License 2.0.
+
+    For more information or inquiries please visit our website at
+    https://www.canyongbs.com or contact us via email at legal@canyongbs.com.
+
+</COPYRIGHT>
+--}}
+@php
+    use League\CommonMark\Extension\Footnote\FootnoteExtension;
+@endphp
+
+<div @if ($this->hasResearchStarted && !$researchRequest?->finished_at) wire:poll.3s @endif>
+    @if (!$researchRequest?->finished_at)
+        <div class="flex items-center gap-2">
+            <x-filament::loading-indicator class="h-5 w-5" /> Researching...
+        </div>
+    @endif
+
+    @if (filled($researchRequest?->results))
+        <section class="prose max-w-none dark:prose-invert">
+            @if (filled($researchRequest->title))
+                <h1>{{ $researchRequest->title }}</h1>
+            @endif
+
+            {!! str($researchRequest->results)->replace('<think>', '<details wire:ignore.self><summary>Reasoning</summary>')->replace('</think>', '</details>')->markdown(extensions: [app(FootnoteExtension::class)]) !!}
```

Comment:
Could someone ask this to generate code examples of XSS and it result in scripts created and executed here?

Reply:
I think Commonmark handles it but I'll sanitise after just to be sure